client/me/account/index.jsx

```diff
@@ -33,7 +33,7 @@ var LanguageSelector = require( 'components/forms/language-selector' ),
 	ReauthRequired = require( 'me/reauth-required' ),
 	twoStepAuthorization = require( 'lib/two-step-authorization' ),
 	user = require( 'lib/user' )(),
-	Notice = require( 'components/notice' ),
+	Notice = require( 'notices/notice' ),
```

Comment:
@johnHackworth : it's good that you pinged me. We are getting completely rid of `components/notice`. 
I think the proper way to do this would be to use `NoticeAction` component.
But yeah, thir is our fault :)